.eslintrc

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
We already have .eslintrc in the `frontend` and `backend`. Repo root is not a js/ts project, so I think it makes more sense to keep per-module conf.